meta/def.rb

```diff
@@ -94,6 +110,7 @@
   mutation 'def foo(_unused = false); end'
   mutation 'def foo(_unused = true); raise; end'
   mutation 'def foo(_unused); end'
+  mutation 'def foo; _unused = true; end'
```

Comment:
Actually what would also make sense is an additional mutation to:

```
def foo(bar, baz = something)
  baz = something
  body
end
```

That would also ensure that the test actually specifies a different than default semantics via the default arguments.